src/Grid/topology.jl

```diff
@@ -263,6 +263,14 @@ function getneighborhood(top::ExclusiveTopology, grid::AbstractGrid, faceidx::Fa
     end
 end
 
+function getneighborhood(top::ExclusiveTopology, grid::AbstractGrid{2}, edgeidx::EdgeIndex, include_self=false)
```

Comment:
I guess it comes down to this
```suggestion
function getneighborhood(top::ExclusiveTopology, grid::AbstractGrid{2,<: Union{<:Ferrite.AbstractCell{<:Ferrite.AbstractRefShape{2}}}}, edgeidx::EdgeIndex, include_self=false)
```